src/components/cap-reporter.js

```diff
@@ -66,6 +66,7 @@ export default class CapReporter extends LitElement {
 				font-weight: 600;
 				font-size: var(--font-size-250);
 				position: relative;
+				margin-bottom: .5rem;
```

Comment:
We can improve internationalization support if we rewrite this margin property using logical properties. We can also incorporate a css variable here that's equivalent to 0.5rem. Combined together, Together, that fix might look like:

`margin-block-end: var(--spacing-75)` 

One additional thing to keep in mind is in what direction you are applying margins. This is more a matter of style — I often apply margins exclusively to one direction (for example, to `margin-block-start`) because it helps me avoid accidentally attempting to the style the same margin space that exists between different components, and otherwise avoids margin collapse, but this isn't really hardfast.

Reply:
A primer on logical properties and how they relate to internationalization support is this blogpost from Sparkbox: https://sparkbox.com/foundry/internationalization_css_html